app.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable linebreak-style */
+// eslint-disable-next-line no-unused-vars
+function togglePopup() {
+  // eslint-disable-next-line indent
+  document.getElementById('popup-1').classList.toggle('active');
+}
+
+// eslint-disable-next-line no-undef
```

Comment:
- [ ] Kindly fix linters instead of disabling them. that could be solved by using event listeners.